docs/tutorials/2_tutorial_hybrid_jobs.ipynb

```diff
@@ -0,0 +1,263 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "adc318c6",
+   "metadata": {},
+   "source": [
+    "# Tutorial: Variational algorithms on Hybrid jobs"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8e36b315",
+   "metadata": {},
+   "source": [
+    "## Hybrid Jobs"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a6c9025e",
+   "metadata": {},
+   "source": [
+    "Amazon Braket Hybrid Jobs offers a way for you to run hybrid quantum-classical algorithms that require both classical resources and quantum processing units (QPUs). Hybrid Jobs is designed to spin up the requested classical compute, run your algorithm, and release the instances after completion so you only pay for what you use. This workflow is ideal for long-running iterative algorithms involving both classical and quantum resources. In this notebook we show you how to run such a job via the qiskit-braket provider.\n",
+    "\n",
+    "Read more at https://docs.aws.amazon.com/braket/latest/developerguide/braket-jobs.html"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5f46c9fe",
+   "metadata": {},
+   "source": [
+    "## Prepare Files"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3f2c43fc",
+   "metadata": {},
+   "source": [
+    "### Prepare Hybrid Job Script"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "18a4937a",
+   "metadata": {},
+   "source": [
+    "The job script defines the specific quantum logic to be executed on the quantum processor and the classical processing that the output is saved to, in this case, setting up the device backend and saving the job results to the EC2 instance.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "b9184273",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\"\"\"Example of Hybrid Job payload with VQE.\"\"\"\r\n",
```

Comment:
Shall we explain somewhere at the top what exactly a VQE is, why it needs hybrid jobs. Then at the bottom we can explain what these results mean.